sdk/resources/arm-resources/README.md

```diff
@@ -1,109 +1,91 @@
-## Azure ResourceManagementClient SDK for JavaScript
+# Azure ResourceManagement client library for JavaScript
 
-This package contains an isomorphic SDK (runs both in Node.js and in browsers) for ResourceManagementClient.
+This package contains an isomorphic SDK (runs both in Node.js and in browsers) for Azure ResourceManagement client.
+
+Provides operations for working with resources and resource groups.
+
+[Source code](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/resources/arm-resources) |
+[Package (NPM)](https://www.npmjs.com/package/@azure/arm-resources) |
+[API reference documentation](https://docs.microsoft.com/javascript/api/@azure/arm-resources) |
+[Samples](https://github.com/Azure-Samples/azure-samples-js-management)
```

Comment:
@sarangan12 we are moving to do this in Track 2 readme so please implement my suggestion in code gen:

```suggestion
Key links:
- [Source code](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/resources/arm-resources)
- [Package (NPM)](https://www.npmjs.com/package/@azure/arm-resources)
- [API reference documentation](https://docs.microsoft.com/javascript/api/@azure/arm-resources)
- [Samples](https://github.com/Azure-Samples/azure-samples-js-management)
```

Reply:
@sarangan12 Please pay attention to the url of Samples. In the PR, I change it to a valid one, but it's invalid in the generated codes of codegen.

Reply:
@dw511214992 we already have https://github.com/Azure/autorest.typescript/issues/1009 open to track this issue.

Reply:
Code changes complete with https://github.com/Azure/autorest.typescript/pull/1098/files#diff-b8b883443a183080ec09c36e77356c30549ddff6b5e7b4ba5fe8285e9030c82bR107